src/WooCommerce.php

```diff
@@ -109,7 +109,7 @@ public static function addCouponAmount($label, $coupon) {
       $amount = wc_price($amount);
     }
 
-    $label .= '<br/>' . sprintf(__('Coupon value: %s', Plugin::L10N), $amount);
+    $label .= sprintf(__('Coupon value: %s', Plugin::L10N), wp_strip_all_tags($amount));
```

Comment:
I understand the need of `wp_strip_all_tags`, but why do you remove the `<br />`? Do you want this string to be displayed on the same line?